src/fprime_gds/executables/run_deployment.py

```diff
@@ -174,11 +228,22 @@ def main():
     # Launch the desired GUI package
     launchers.append(launch_html)
 
+    # Check if OpenMCT is set to be used. If true, generate OpenMCT States and Launch the OpenMCT Server
+    if parsed_args.openmct:
+        # Try Generating the OpenMCT JSON and Initial States
+        get_openmct_json(parsed_args)
+
+        # Try Launching OpenMCT Server
+        launchers.append(launch_openmct)
+
+        #Try polling for telemetry
+        launchers.append(poll_telem)
+
     # Launch launchers and wait for the last app to finish
     try:
         procs = [launcher(parsed_args) for launcher in launchers]
         print("[INFO] F prime is now running. CTRL-C to shutdown all components.")
-        procs[-1].wait()
+        procs[-3].wait()
```

Comment:
Why do the last 2 processes need to stay alive? We will likely want to avoid doing that as it impacts all other deployment runs, but if that's really needed for openmct you can just have `if parsed_args.openmct:` that allows for it.

Reply:
they do not need to stay alive, I reverted back to the original ```procs[-1].wait()```